src/main/java/net/minecraftforge/client/event/RenderUnderwaterOverlayEvent.java

```diff
@@ -0,0 +1,15 @@
+package net.minecraftforge.client.event;
+
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+import net.minecraftforge.fml.common.eventhandler.Event;
+
+/**
+ * This event is fired before the underwater overlay is rendered
+ * This event is {@link Cancelable}
+ * If canceled, the underwater overlay will not be rendered.
+ * This event is fired from {@link net.minecraft.client.renderer.ItemRenderer#renderWaterOverlayTexture(float)}
+ */
+@Cancelable
+public class RenderUnderwaterOverlayEvent extends Event
+{
```

Comment:
I don't see anything useful available from where the event is fired that's not already available as a singleton.